Decisions.Microsoft365.Exchange/InputExchangeSettings.cs

```diff
@@ -0,0 +1,55 @@
+using System.ComponentModel;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+using DecisionsFramework.Design.Properties;
+using DecisionsFramework.Design.Properties.Attributes;
+
+namespace Decisions.Microsoft365.Exchange;
+
+public class InputExchangeSettings: ICreateChildEntityInFolder
+{
+    private string graphUrl = "https://graph.microsoft.com/v1.0";
+
+    [PropertyClassification(0, "Graph URL", "Exchange Settings")]
+    [WritableValue]
+    public string GraphUrl
```

Comment:
Once we update this to be an `ORMEntity`, we need to add `DataMember` attributes to all of these.